recipes/example/meta.yaml

```diff
@@ -21,6 +21,8 @@ source:
 build:
   number: 0
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py < 34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
```

Comment:
Yeah, to be honest, I'm not sure conda-forge needs more reminders about Windows. We are probably more aware than most of its value to community and everyone has made a decent effort chipping in.

That being said, building for Windows with complicated packages is not an easy ask. It may involve working with a Windows expert. So it is not uncommon for people to at least delay them until there is a feedstock and then spend time trying to address the issue. This is good for technical and organizational reasons.

In any event, as mentioned non-blocker and am glad we have some Python and OS skips in the comments. Nice addition. 👍